src/Nethermind/Nethermind.Evm/Tracing/GethStyle/GethLikeTxTrace.cs

```diff
@@ -7,9 +7,16 @@
 
 namespace Nethermind.Evm.Tracing.GethStyle;
 
-public class GethLikeTxTrace
+public class GethLikeTxTrace : IDisposable
 {
-    public GethLikeTxTrace() => Entries = new List<GethTxTraceEntry>();
+    private readonly IDisposable? _disposable;
+
+    public GethLikeTxTrace(IDisposable? disposable = null)
```

Comment:
Can we please make what we pass here more apparent? Or is it deliberate?

Reply:
As the serialization is lazy and we need to dispose of something AFTER serialization, we pass it here as disposable.

Reply:
Maybe the variable name could be clearer, what do we need to dispose of after serialisation?

Reply:
We are disposing V8Engine here that was used to run and build javascript types. We cannot dispose it before serialization, as some types that will be serialized are connected to it.